internal/cri/server/service.go

```diff
@@ -108,6 +108,8 @@ type ImageService interface {
 	LocalResolve(refOrID string) (imagestore.Image, error)
 
 	ImageFSPaths() map[string]string
+
+	Config() criconfig.ImageConfig
```

Comment:
I named this one as `Config()` to be consistent with `RuntimeService.Config()`, but it does cause conflict within `criService`

https://github.com/containerd/containerd/blob/bc3ce87c8475e67572ab14b96501acefb9799307/internal/cri/server/service.go#L115-L116

Maybe we should name this as `ImageConfig() criconfig.ImageConfig`? So `criService.Config()` still returns `criconfig.Config` same as before, and `criService.ImageConfig()` returns the `criconfig.ImageConfig`